### DISCUSSION TITLE
Welcome to nft-marketplace Discussions!

### DISCUSSION THREAD

**Discussion by yessGlory17** [yessGlory17/nft-marketplace#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abdiisadisdnskx** [comment]
When is the alpha release?

**Reply by yessGlory17** [reply]
I am currently in a very busy exam period. I'm going to post some commits tonight. But the Alpha version will probably be ready next week.

**Reply by sushilraghava** [reply]
Hi, i hope you did very well in your exams and now ready to help me to check your NFT marketplace demo somewhere or you can install it on my server ? Can you do it Elrond based ? My skype is sushilraghav
